code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -288,14 +288,16 @@
 	var/responses = jointext(GLOB.syndicate_code_response, ", ")
 
 	to_chat(traitor_mob, "<U><B>The Syndicate have provided you with the following codewords to identify fellow agents:</B></U>")
-	to_chat(traitor_mob, "<B>Code Phrase: <span class='danger'>[phrases]</span></B>")
-	to_chat(traitor_mob, "<B>Code Response: <span class='danger'>[responses]</span></B>")
+	to_chat(traitor_mob, "<B>Code Phrase: <span class='codephrases'>[phrases]</span></B>")
+	to_chat(traitor_mob, "<B>Code Response: <span class='coderesponses'>[responses]</span></B>")
 
 	antag_memory += "<b>Code Phrase</b>: <span class='red'>[phrases]</span><br>"
 	antag_memory += "<b>Code Response</b>: <span class='red'>[responses]</span><br>"
 
 	to_chat(traitor_mob, "Use the codewords during regular conversation to identify other agents. Proceed with caution, however, as everyone is a potential foe.")
+	to_chat(traitor_mob, "<b><font color=red>You memorize the codewords, allowing you to recognize them when heard.</font></b>")
 
+	traitor_mob.client.chatOutput?.notify_syndicate_codes()
```

Comment:
Can you add a method to force clear the chat highlights, for cases such as admins de-traitoring someone?